Swift/DocumentModel.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  DocumentModel.swift
```

Comment:
Probably this file is created when we still have the DocumentModel macro? Should the file be DocumentID.swift as the only class here is DocumentID?